catalog.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>HTML Academy: Седона</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <a class="logo" href="index.html">
+          <img src="https://via.placeholder.com/138x70" width="138" height="70" alt="Логотип городка Седона">
+        </a>
+        <ul>
+          <li><a href="">Информация</a></li>
+          <li><a href="">Фото и видео</a></li>
+          <li><a href="">Карта штата</a></li>
+          <li><a href="catalog.html">Гостиницы</a></li>
```

Comment:
Страница не должна ссылаться на себя (чтобы не вводить пользователя в заблуждение). Для этого просто убираем `href` у ссылки:
```suggestion
          <li><a>Гостиницы</a></li>
```

Reply:
Готово.